clients/rust/tests/setup/mod.rs

```diff
@@ -141,9 +141,9 @@ pub fn calculate_stake_rewards_per_token(rewards: u64, stake_amount: u64) -> u12
     } else {
         // Calculation: rewards / stake_amount
         //
-        // Scaled by 1e9 to store 9 decimal places of precision.
+        // Scaled by 1e18 to store 18 decimal places of precision.
         (rewards as u128)
-            .checked_mul(1_000_000_000)
+            .checked_mul(1_000_000_000_000_000_000)
```

Comment:
nit: should we use the constant here?
```suggestion
            .checked_mul(REWARDS_PER_TOKEN_SCALING_FACTOR)
```

Reply:
Yeah sure!